package.json

```diff
@@ -7,13 +7,11 @@
   "repository": "gulpjs/eslint-config-gulp",
   "license": "MIT",
   "engines": {
-    "node": ">= 0.10"
+    "node": ">=10"
   },
   "main": "eslint.js",
   "files": [
-    "LICENSE",
```

Comment:
I know that node automatically injects LICENSE, but I like to keep it here for to be very specific. If the defaults ever changed, we would absolutely need the license to be shipped (opposed to the README).